arm-datalake-store/account/2016-11-01/swagger/account.json

```diff
@@ -242,7 +295,7 @@
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the Data Lake Store account to which to add the trusted identity provider."
+            "description": "The name of the Data Lake Store account to which to add or replace the trusted identity provider."
```

Comment:
I would like to keep the add/replace description, since it will replace the rule if it already exists. I took out the "to which" though, since it adds a lot of extra verbage :)